src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,8 @@
 // Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
```

Comment:
React fragment we use to return multiple Components/Elements without parent one.
```jsx
function App() {
    return (
      <>
          <ComponentA/>
          <ComponentB/>
          <ComponentC/>
      </>
    );
  }
```